KS.Fiks.IO.Client.Tests/Send/SendHandlerTests.cs

```diff
@@ -54,7 +55,35 @@ public async Task CallsSendWithExpectedMessageSpecificationApiModel()
                     model => model.Ttl == (long)request.Ttl.TotalMilliseconds &&
                              model.SvarPaMelding == request.SvarPaMelding &&
                              model.AvsenderKontoId == request.AvsenderKontoId &&
-                             model.MottakerKontoId == request.MottakerKontoId),
+                             model.MottakerKontoId == request.MottakerKontoId &&
+                             model.Headere["klientMeldingId"] == request.KlientMeldingId.ToString()),
+                It.IsAny<Stream>()));
+        }
+
+        [Fact]
+        public async Task CallsSendWithExpectedMessageSpecificationApiModelAndNullKlientMeldingId()
+        {
+            var sut = _fixture.CreateSut();
+
+            var request = new MeldingRequest(
+                avsenderKontoId: Guid.NewGuid(),
+                mottakerKontoId: Guid.NewGuid(),
+                meldingType: "Meldingsprotokoll",
+                ttl: TimeSpan.FromDays(2),
+                headere: null,
+                svarPaMelding: Guid.NewGuid());
+
+            var payload = new List<IPayload>();
+
+            await sut.Send(request, payload).ConfigureAwait(false);
+
+            _fixture.FiksIOSenderMock.Verify(_ => _.Send(
+                It.Is<MeldingSpesifikasjonApiModel>(
+                    model => model.Ttl == (long)request.Ttl.TotalMilliseconds &&
+                             model.SvarPaMelding == request.SvarPaMelding &&
+                             model.AvsenderKontoId == request.AvsenderKontoId &&
+                             model.MottakerKontoId == request.MottakerKontoId &&
+                             !model.Headere.ContainsKey("klientMeldingId")),
```

Comment:
Bruk konstant i stedet for String litteral

Reply:
damnit
Glemte den 1 sted ja. Good catch!